aio/content/examples/getting-started/src/app/cart.service.ts

```diff
@@ -2,13 +2,14 @@
 // #docregion import-http
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+// #docregion props
 import { Product } from './products';
 // #enddocregion import-http
```

Comment:
Since we only want to show the relevant bits in the `props` docregion, I think it would be better to end the `props` docregion here (and revert the change on line 12):

```suggestion
// #enddocregion import-http, props
```